lib/authentication/AuthenticationService.ts

```diff
@@ -43,6 +43,8 @@ export class AuthenticationService {
 
       if (user && !user.isLocked()) {
         if (user.party.code === code) {
+          user.failedAuthAttempts = 0;
+          user.save();
```

Comment:
Is the promise intended to be unused or does this happen by accident? If its intended, do we need to comment this to clarify that?

Reply:
It is intended. I though, it is not necessary to wait on the completion at this point. Do you think, we should wait here?

Reply:
No, I don't think so. But should we mention this in the code via comment?